src/database.py

```diff
@@ -0,0 +1,215 @@
+import os
+import logging
+from sqlalchemy import inspect, create_engine, Sequence, Column, Integer, String, Float, DateTime
+from sqlalchemy.orm import sessionmaker, declarative_base
+import polars as pl
+
+# Set up logging
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(__name__)
+
+# Define the base for ORM models
+Base = declarative_base()
+
+# ORM model for the 'orders' table
+class Order(Base):
```

Comment:
For a ll models kindly add created_at and updated_at, good to keep track of the time your data enters into the db. This is the bedrock of time based queries you'd make in the future